docs/specification.md

```diff
@@ -90,14 +91,18 @@ In CENTRE they take a single item (a pair or an amount).
 5. `configure_minter` takes an additional argument – current minting allowance – to prevent front-running attacks.
 6. CENTRE does not have `update_operator`, the closest analog is `approve`, but it does not explicitly say whether `approve` is paused by `pause`.
 In this contract `update_operator` IS paused.
-7. TODO: whitelisting and blacklisting.
+7. CENTRE defines the `blacklister` role and blacklisting is part of the CENTRE Fiat Token contract.
+Our project is different: whitelisting and blacklisting are offloaded to a separate [`Safelist` contract](#safelist).
```

Comment:
Isn't this kind of functionality meant to be implemented by the "transfer_hook" FA2 spec and "set_transfer_hook" infra?

Reply:
It's not specified whether we will implement a monolithic contract or a modular one with a transfer hook. If we implement a monolithic one, it will call safelist contract directly. If we implement a modular one, it will call the transfer hook which will in turn call the safelist contract.

Reply:
I get it now.